WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -161,12 +161,15 @@ public static void browsePostOrPage(Context context, Blog blog, Post post) {
         if (blog == null || post == null || TextUtils.isEmpty(post.getPermaLink())) return;
 
         String url = post.getPermaLink();
-        if (-1 == url.indexOf('?')) {
-            url = url.concat("?preview=true");
-        } else {
-            url = url.concat("&preview=true");
+        // Add the preview=true parameter if the post is not published yet
+        if (!post.isPublished()) {
```

Comment:
Hm, I would like a "isRemoteChangeNotPublished()" ;) - I think that's actually not possible